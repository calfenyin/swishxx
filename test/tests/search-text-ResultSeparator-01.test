search | search-ResultSeparator-01.conf | | year | 0
