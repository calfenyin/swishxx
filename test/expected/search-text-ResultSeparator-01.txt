# results: 3
100>./Christmas_Carol,_A.txt>162261>Christmas_Carol,_A.txt
95>./Alice's_Adventures_in_Wonderland.txt>147773>Alice's_Adventures_in_Wonderland.txt
76>./Time_Machine,_The.txt>182203>Time_Machine,_The.txt
